workflow/rules/sbs_process.smk

```diff
@@ -137,3 +137,168 @@ rule apply_ic_field:
+    input:
+        SBS_PROCESS_FP
+        / "tsvs"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "bases", "tsv"),
+        SBS_PROCESS_FP
+        / "images"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "peaks", "tiff"),
+    output:
+        SBS_PROCESS_FP
+        / "tsvs"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "reads", "tsv"),
+    script:
+        "../scripts/sbs_process/call_reads.py"
+
+
+# Call cells
+rule call_cells:
+    conda:
+        "../envs/sbs_process.yml"
+    input:
+        SBS_PROCESS_FP
+        / "tsvs"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "reads", "tsv"),
+    output:
+        SBS_PROCESS_FP
+        / "tsvs"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "cells", "tsv"),
+    params:
+        df_design_path=config["sbs_process"]["df_design_path"],
+        q_min=config["sbs_process"]["q_min"],
+    script:
+        "../scripts/sbs_process/call_cells.py"
+
+
+# Extract minimal phenotype features
+rule extract_phenotype_minimal:
+    conda:
+        "../envs/sbs_process.yml"
+    input:
+        SBS_PROCESS_FP
+        / "images"
+        / get_filename({"well": "{well}", "tile": "{tile}"}, "nuclei", "tiff"),
+    output:
+        SBS_PROCESS_FP
+        / "tsvs"
+        / get_filename(
+            {"well": "{well}", "tile": "{tile}"}, "minimal_phenotype_info", "tsv"
+        ),
+    script:
+        "../scripts/sbs_process/extract_phenotype_minimal.py"
+
+
+# Rule for combining read results from different wells
+rule merge_reads:
+    conda:
+        "../envs/sbs_process.yml"
+    input:
+        lambda wildcards: expand(
+            SBS_PROCESS_FP
+            / "tsvs"
+            / get_filename({"well": wildcards.well, "tile": "{tile}"}, "reads", "tsv"),
+            tile=SBS_TILES,
+        ),
+    output:
+        SBS_PROCESS_FP / "hdfs" / get_filename({"well": "{well}"}, "reads", "hdf5"),
+    script:
+        "../scripts/shared/merge_dfs.py"
+
+
+# Rule for combining cell results from different wells
+rule merge_cells:
+    conda:
+        "../envs/sbs_process.yml"
+    input:
+        lambda wildcards: expand(
+            SBS_PROCESS_FP
+            / "tsvs"
+            / get_filename({"well": wildcards.well, "tile": "{tile}"}, "cells", "tsv"),
+            tile=SBS_TILES,
+        ),
+    output:
+        SBS_PROCESS_FP / "hdfs" / get_filename({"well": "{well}"}, "cells", "hdf5"),
+    script:
+        "../scripts/shared/merge_dfs.py"
+
+
+# Rule for combining phenotypic info results from different wells
+rule merge_minimal_phenotype_info:
+    conda:
+        "../envs/sbs_process.yml"
+    input:
+        lambda wildcards: expand(
+            SBS_PROCESS_FP
+            / "tsvs"
+            / get_filename(
+                {"well": wildcards.well, "tile": "{tile}"},
+                "minimal_phenotype_info",
+                "tsv",
+            ),
+            tile=SBS_TILES,
+        ),
```

Comment:
Unfortunately, we use the wording "merge" as a step in the pipeline. So I would start changing all of these and their underlying functions to "combine"

eg combine_minimal_phenotype_info

Reply:
Ah, good point. Done in https://github.com/cheeseman-lab/brieflow/pull/8/commits/3f34212f7984650a2ff4e43cae8f7648e89cdfc0